torchreid/losses/am_softmax.py

```diff
@@ -174,45 +174,56 @@ def forward(self, cos_theta, target, aug_index=None, lam=None, iteration=None):
                 Each position contains the label index.
             iteration (int): current iteration
         """
+        if (self.aug_type and aug_index is not None and lam is not None):
+            targets1 = torch.zeros(cos_theta.size()).scatter_(1, target.unsqueeze(1).data.cpu(), 1)
```

Comment:
Why do you need implicitly call cpu() here? If `target` was a cuda tensor, it will cause slowdown.